src/client/containers/app.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import CreateList from '../components/createlist';
+import TodoList from '../components/todolist';
+import Title from '../components/title';
+import Refresh from '../components/refresh';
+import Spinner from '../components/spinner';
+import { createList, fetchList, removeList } from '../actions/lists';
+import { createTask, removeTask } from '../actions/tasks';
+import { sortAlphaListSelector, sortAlphaTaskSelector } from '../selectors';
+import { connect } from 'react-redux';
+
+class TodoApp extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentWillMount = () => {
```

Comment:
generally, it's better to hit the server once we are sure to be able to display somethin'
using didMount may be a better choice